src/Microsoft.DotNet.Build.Tasks.net45/Microsoft.DotNet.Build.Tasks.net45.csproj

```diff
@@ -39,4 +39,25 @@
     <TargetingPackReference Include="System.Xml" />
     <TargetingPackReference Include="System.Xml.Linq" />
   </ItemGroup>
+  <Target Name="AfterBuild">
+    <!-- Manually binplace DiaSymReader runtimes for win7-x86, which the official builds use. -->
```

Comment:
Oh, I mistook what this was a comment about. I'll be adding issues for this hack and Core support.